mockgen/reflect.go

```diff
@@ -41,8 +41,13 @@ func Reflect(importPath string, symbols []string) (*model.Package, error) {
 
 	progPath := *execOnly
 	if *execOnly == "" {
+		// Put program in working directory to pick up packages in vendor/ during build
+		wDir, err := os.Getwd()
```

Comment:
This is a non-starter for anyone using this where the source files might be read-only. We might need to resort to this only when needed (e.g. if there's vendoring involved).